pkgs/os-specific/linux/targetcli/default.nix

```
@@ -2,17 +2,22 @@
 
 python.pkgs.buildPythonApplication rec {
   pname = "targetcli";
-  version = "2.1.fb49";
+  version = "2.1.50";
 
   src = fetchFromGitHub {
     owner = "open-iscsi";
     repo = "${pname}-fb";
     rev = "v${version}";
-    sha256 = "093dmwc5g6yz4cdgpbfszmc97i7nd286w4x447dvg22hvwvjwqhh";
+    sha256 = "0xrvby63i39rvi778bnvnxacghaix63q72vzxdc3i87ji1ki58hc";
   };
 
   propagatedBuildInputs = with python.pkgs; [ configshell rtslib ];
 
+  postInstall = ''
+    mkdir -p $out/share/man/man8
+    cp targetcli.8 $out/share/man/man8/
```

```suggestion
    install -D targetcli.8 -t $out/share/man/man8/
```
And remove the line above.